natlas-agent/setup-agent.sh

```diff
@@ -83,6 +83,14 @@ else
 	echo "[+] Found nmap: $(which nmap)"
 fi
 
+echo "[+] Setting capabilities for nmap"
+if ! setcap cap_net_raw,cap_net_admin,cap_net_bind_service+eip /usr/bin/nmap >/dev/null 2>&1; then
```

Comment:
This hardcodes the location of the nmap binary. Can you replace it with $(which nmap) to be more robust?

Reply:
fixed

Reply:
NIT: command -v is the right way according to my linter.